src/components/SignUpSellerForm/SignUpSellerForm.jsx

```diff
@@ -0,0 +1,302 @@
+        phone: { valid: true },
+      }))
+
+      return
+    }
+
+    setErrors((errors) => ({
+      ...errors,
+      phone: {
+        valid: false,
+        errorMessage: 'Вкажіть у форматі ХХХ ХХХ ХХ ХХ',
+      },
+    }))
+  }, [phone])
+
+  function formatPhoneNumber(phone) {
+    const cleanNum = ('' + phone.toString()).replace(/\D/g, '')
+    const match = cleanNum.match(/^(\d{3})(\d{3})(\d{2})(\d{2})$/)
+    if (match) {
+      return [
+        '+38 ',
+        '(',
+        match[1],
+        ') ',
+        match[2],
+        '-',
+        match[3],
+        '-',
+        match[4],
+      ].join('')
+    }
+    return cleanNum
+  }
+
+  const validatePassword = useCallback(() => {
+    const pattern = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{9,}$/
+    if (pattern.test(password)) {
+      setErrors((errors) => ({
+        ...errors,
+        password: { valid: true },
+      }))
+
+      return
+    }
+
+    setErrors((errors) => ({
+      ...errors,
+      password: {
+        valid: false,
+        errorMessage: 'Від 8 літер та 1 числовий символ',
+      },
+    }))
+  }, [password])
+
+  const handleFullNameChange = (event) => {
+    const newFullName = event.target.value
+    setFullName(newFullName)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      fullName: newFullName,
+    }))
+  }
+
+  const handleCompanyNameChange = (event) => {
+    const newCompanyName = event.target.value
+    setCompanyName(newCompanyName)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      companyName: newCompanyName,
+    }))
+  }
+
+  const handleEmailChange = (event) => {
+    const newEmail = event.target.value
+    setEmail(newEmail)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      email: newEmail,
+    }))
+  }
+
+  const handlePhoneChange = (event) => {
+    const newPhone = event.target.value
+    setPhone(newPhone)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      phone: formatPhoneNumber(newPhone),
+    }))
+  }
+
+  const handlePasswordChange = (event) => {
+    const newPassword = event.target.value
+    setPassword(newPassword)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      password: newPassword,
+    }))
+  }
+
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false)
+  const handleEyeButton = () => {
+    setIsPasswordVisible(!isPasswordVisible)
+  }
+
+  useEffect(() => {
+    const hasErrors = Object.values(errors).some(({ valid }) => valid === false)
+
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      hasErrors,
+    }))
+  }, [errors])
+
+  return (
+    <div className={styles.formContainer}>
+      <div className={styles.inputsWrapper}>
+        {' '}
+        <label htmlFor="fullName">ПІБ</label>
+        <input
+          type="text"
+          id="fullName"
+          name="fullName"
+          placeholder="Василь Іванович Глушко"
+          minLength="3"
+          // maxLength="50"
```

Comment:
we should wither remove unused attributes or uncomment them and set the required values for them.

Reply:
ok